frontend/package.json

```diff
@@ -7,7 +7,7 @@
     "@fortawesome/free-brands-svg-icons": "^5.15.3",
     "@fortawesome/free-solid-svg-icons": "^5.15.3",
     "@fortawesome/react-fontawesome": "^0.1.14",
-    "axios": "^0.19.0",
+    "axios": "^0.21.1",
```

Comment:
i wonder if this will cause any problems? the axios is updated to a newer version. Anyone know for sure? A bit concerned because all of our requests will be affected.